hasura/migrations/default/1652824553996_alter_table_public_vaults_alter_column_vault_address/up.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."vaults" add constraint "vaults_vault_address_key" unique ("vault_address");
```

Comment:
should the unique key be `address` + `org_id` ?

Reply:
@zashton I considered this, but then we could have a situation where a separate org is somehow using the same vault. I don't think that's desirable right now.